### PR TITLE
Add NL formatting rules

### PR DESCRIPTION
**Pull request by Pitchiu**
This PR adds formatting rules for the Netherlands.
Sources:
- https://nl.wikipedia.org/wiki/Huisnummer
- https://nl.wikipedia.org/wiki/Adres_(woonplaats)

### CODE REVIEW

**Comment by Pitchiu** [comment]
@norgevz could you take a look?

**Review by skyman503 — APPROVED** [review]
Lgtm thanks!

File: model/countries/NL/NL-formatting-rules.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  - street-address-alternative-1
+  - separator: "\n"
+  - postal-code
+  - separator: " "
+  - locality1
+  - separator: "\n"
+  - country-name
+  - skip: country  # redundant with country-name
+  - skip: street-address  # redundant with street-address-alternative-1
+  - skip: admin-area1 # not commonly used
```
Comment (norgevz) [inline model/countries/NL/NL-formatting-rules.yaml:12]: Could you add to the PR description the main sources that you used to build these expressions? I'm also interested in the following questions:

1. Is okay to always use `-` as the separator building and unit?
2. Is state (admin-area1) really not used in the Netherlands?

**Review by norgevz — CHANGES_REQUESTED** [review]
LGTM

Thanks!

**Review by norgevz — APPROVED** [review]

**Pull request merged**